routes/ProductionRequest.ts

```diff
@@ -0,0 +1,121 @@
+  statusUpdateEmails: Joi.array().items(Joi.string()).required(),
+  valueProvided: Joi.string().required(),
+  businessModel: Joi.string(),
+  policyDocuments: Joi.array().items(Joi.string()).required(),
+  phoneNumber: Joi.string().required(),
+  apis: Joi.custom(validateApiList).required(),
+  monitizedVeteranInformation: Joi.boolean().required(),
+  monitizationExplanation: Joi.string(),
+  veteranFacing: Joi.boolean(),
+  website: Joi.string(),
+  signUpLink: Joi.string(),
+  supportLink: Joi.string(),
+  platforms: Joi.array().items(Joi.string()),
+  veteranFacingDescription: Joi.string(),
+  vasiSystemName: Joi.string(),
+  credentialStorage: Joi.string().required(),
+  storePIIOrPHI: Joi.boolean().required(),
+  storageMethod: Joi.string(),
+  safeguards: Joi.string(),
+  breachManagementProcess: Joi.string(),
+  vulnerabilityManagement: Joi.string(),
+  exposeHealthInformationToThirdParties: Joi.boolean(),
+  thirdPartyHealthInfoDescription: Joi.string(),
+  scopesAccessRequested: Joi.array().items(Joi.string()),
+  distrubitingAPIKeysToCustomers: Joi.boolean(),
+  namingConvention: Joi.string(),
+  centralizedBackendLog: Joi.string(),
+  listedOnMyHealthApplication: Joi.boolean(),
+}).options({ abortEarly: false });
+
+interface ContactDetails {
+  firstName: string;
+  lastName: string;
+  email: string;
+}
+
+export interface ProductionAccessConsumerEmail {
+  logo: string;
+  stepOne: string;
+  stepTwo: string;
+  stepThree: string;
+  stepFour: string;
+}
+interface ProductionAccessBody {
+  primaryContact: ContactDetails;
+  secondaryContact: ContactDetails;
+  organization: string;
+  appName: string;
+  appDescription: string;
+  statusUpdateEmails: string[];
+  valueProvided: string;
+  businessModel?: string;
+  policyDocuments: string[];
+  phoneNumber: string;
+  apis?: string;
+  monitizedVeteranInformation: boolean;
+  monitizationExplanation?: string;
+  veteranFacing?: boolean;
+  website?: string;
+  signUpLink?: string;
+  supportLink?: string;
+  platforms?: string[];
+  veteranFacingDescription?: string;
+  vasiSystemName?: string;
+  credentialStorage: string;
+  storePIIOrPHI: boolean;
+  storageMethod?: string;
+  safeguards?: string;
+  breachManagementProcess?: string;
+  vulnerabilityManagement?: string;
+  exposeHealthInformationToThirdParties?: boolean;
+  thirdPartyHealthInfoDescription?: string;
+  scopesAccessRequested?: string[];
+  distrubitingAPIKeysToCustomers?: boolean;
+  namingConvention?: string;
+  centralizedBackendLog?: string;
+  listedOnMyHealthApplication?: boolean;
+}
+type ProductionAccessRequest = Request<Record<string, unknown>, Record<string, unknown>, ProductionAccessBody, Record<string, unknown>>;
+
+export default function productionRequestHandler( govdelivery: GovDeliveryService | undefined) {
+  return async function (
+    req: ProductionAccessRequest,
+    res: Response,
+    next: NextFunction
+  ): Promise<void> {
+    try {
+      if (govdelivery) {
```

Comment:
hate that this is still a thing but whatever, now's probably not the right time to worry about that